examples/simple/implementation-defined/IMPL-CCV2-bad-static.c

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2015 RV-Match Team. All Rights Reserved.
+// Copyright (c) 2015 Runtime Verification, Inc. All Rights Reserved.
```

Comment:
The copyright should be as discussed by email:

// Copyright (c) 2015 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.